cmd/loki-curator/app/loki_curator.go

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"flag"
+	"net/http"
+	"os"
+	"time"
+
+	lokicurator "github.com/gardener/logging/pkg/loki-curator"
+	config "github.com/gardener/logging/pkg/loki-curator/config"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"github.com/gardener/logging/pkg/loki-curator/metrics"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/weaveworks/common/logging"
+)
+
+var (
+	logger log.Logger
+)
+
+// LokiCuratorInit initializes the configurations
+func LokiCuratorInit() (*config.CuratorConfig, error) {
+	// metrics
+	go func() {
+		http.Handle("/curator/metrics", promhttp.Handler())
+		_ = http.ListenAndServe(":3101", nil)
+	}()
+
+	yamlPath := flag.String("config", "/etc/loki/curator.yaml", "A path to the curator's configuration file")
+	flag.Parse()
+	conf, err := config.ParseConfigurations(*yamlPath)
+	if err != nil {
+		return nil, err
+	}
+
+	logger = newLogger(conf.LogLevel)
+	level.Info(logger).Log("LogLevel", conf.LogLevel)
+	level.Info(logger).Log("TriggerInterval", conf.TriggerInterval)
+	level.Info(logger).Log("DiskPath", conf.DiskPath)
+	level.Info(logger).Log("MinFreeInodesPercentages", conf.MinFreeInodesPercentages)
+	level.Info(logger).Log("TargetFreeInodesPercentages", conf.TargetFreeInodesPercentages)
+	level.Info(logger).Log("InodesPageSizeForDeletionPercentages", conf.InodesPageSizeForDeletionPercentages)
+	level.Info(logger).Log("MinFreeStoragePercentages", conf.MinFreeStoragePercentages)
+	level.Info(logger).Log("TargetFreeStoragePercentages", conf.TargetFreeStoragePercentages)
+	level.Info(logger).Log("CapacityPageSizeForDeletionPercentages", conf.CapacityPageSizeForDeletionPercentages)
+
+	return conf, nil
+}
+
+func newLogger(logLevelName string) log.Logger {
+	var logLevel logging.Level
+	switch logLevelName {
+	case "info":
+		fallthrough
+	case "debug":
+		fallthrough
+	case "warn":
+		fallthrough
+	case "error":
+		logLevel.Set(logLevelName)
+	default:
+		logLevel.Set("info")
+	}
+
+	logger := log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr))
```

Comment:
This variable is already globally defined
```suggestion
	logger = log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr))
```